plugins/backend/catalog-backend-module-okta/src/providers/OktaUserProvider.ts

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2021 Larder Software Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { UserEntity } from '@backstage/catalog-model';
+import * as winston from 'winston';
+import { Config } from '@backstage/config';
+import { OktaEntityProvider } from './OktaEntityProvider';
+
+/**
+ * Provides entities from Okta User service.
+ */
+export class OktaUserProvider extends OktaEntityProvider {
```

Comment:
I think the convention is to call it an entityprovider
```suggestion
export class OktaUserEntityProvider extends OktaEntityProvider {
```

Reply:
done